nuxt.config.js

```diff
@@ -1,7 +1,16 @@
+// The repo name should match name of repository hosting static project.
+// The repo name will be used as the base URL. Otherwise images won't load.
+const REPO_NAME = '/www/'
```

Comment:
OK for a first commit, but in the future you'll want to figure out how to host directly at the root of gh-pages.

Reply:
Because of the way GitHub hosting works, the root directory is the same as the repo name. The only exception for this is whenever the repo name is the same as the username. This article shows what I mean: [Hosting your website on Github](https://www.khanacademy.org/computing/computer-programming/html-css/web-development-tools/a/hosting-your-website-on-github). Here are some examples:

| Repo Name | Resulting URL | 
| --- | --- |
| www | project-imprimis.github.io/www |
| libprimis | project-imprimis.github.io/libprimis | 
| project-imprimis.github.io | project-imprimis.github.io | 

I believe what you suggested was to host the website on `project-imprimis.github.io` instead of `project-imprimis.github.io/www`. As shown by the table, this is not possible without modifying the repo name. Luckily, a custom domain name can be provided.